build-scripts/rcm-guest/push-openshift-online-to-mirrors.sh

```diff
@@ -22,7 +22,7 @@ if [ "$BUILD_MODE" == "online:int" ]; then
 	REPO="online-openshift-scripts-int"
 elif [ "$BUILD_MODE" == "online:stg" ]; then
 	REPO="online-openshift-scripts-stg"
-elif [ "$BUILD_MODE" == "release" -o "$BUILD_MODE" == "pre-release" ]; then
+elif [ "$BUILD_MODE" == "release" ] || [ "$BUILD_MODE" == "pre-release" ]; then
```

Comment:
If you do not need to use `#!/bin/sh` or POSIX compliance... use `[[` as it's going to make this less annoying:

```sh
elif [[ "$BUILD_MODE" == "release" || "$BUILD_MODE" == "pre-release" ]]; then
```